src/customer/strategies/googlepay/googlepay-authorizenet-customer-strategy.spec.ts

```diff
@@ -0,0 +1,258 @@
+
+            it('fails to initialize the strategy if no methodid is supplied', () => {
+                customerInitializeOptions = getAuthNetCustomerInitializeOptions(Mode.UndefinedMethodId);
+
+                function initialize() {
+                    strategy.initialize(customerInitializeOptions);
+                }
+
+                expect(initialize).toThrow(InvalidArgumentError);
+            });
+
+            it('fails to initialize the strategy if no valid container id is supplied', () => {
+                customerInitializeOptions = getAuthNetCustomerInitializeOptions(Mode.InvalidContainer);
+
+                return expect(strategy.initialize(customerInitializeOptions)).rejects.toThrow(InvalidArgumentError);
+            });
+        });
+    });
+
+    describe('#deinitialize()', () => {
+        let customerInitializeOptions: CustomerInitializeOptions;
+
+        beforeEach(() => {
+            customerInitializeOptions = getAuthNetCustomerInitializeOptions();
+        });
+
+        it('succesfully deinitializes the strategy', async () => {
+            await strategy.initialize(customerInitializeOptions);
+
+            strategy.deinitialize();
+
+            if (customerInitializeOptions.googlepayauthorizenet) {
+                const button = document.getElementById(customerInitializeOptions.googlepayauthorizenet.container);
+
+                if (button) {
+                    expect(button.firstChild).toBe(null);
+                }
+            }
+
+            // Prevent "After Each" failure
+            container = document.createElement('div');
+            document.body.appendChild(container);
+        });
+
+        it('Validates if strategy is loaded before call deinitialize', async () => {
+            await strategy.deinitialize();
+
+            if (customerInitializeOptions.googlepayauthorizenet) {
+                const button = document.getElementById(customerInitializeOptions.googlepayauthorizenet.container);
+
+                if (button) {
+                    expect(button.firstChild).toBe(null);
+                }
+            }
+
+            // Prevent "After Each" failure
+            container = document.createElement('div');
+            document.body.appendChild(container);
+        });
+    });
+
+    describe('#signIn()', () => {
+
+        it('throws error if trying to sign in programmatically', async () => {
+            customerInitializeOptions = getAuthNetCustomerInitializeOptions();
+
+            await strategy.initialize(customerInitializeOptions);
+
+            expect(() => strategy.signIn({ email: 'foo@bar.com', password: 'foobar' })).toThrowError();
+        });
+    });
+
+    describe('#signOut()', () => {
+        beforeEach(async () => {
+            customerInitializeOptions = getAuthNetCustomerInitializeOptions();
+
+            await strategy.initialize(customerInitializeOptions);
+        });
+
+        it('throws error if trying to sign out programmatically', async () => {
```

Comment:
I can't seem to find any error being thrown in the test assertions? Not sure if the test title is wrong or you are missing to assert the error